linux-cachyos-hardened/PKGBUILD

```diff
@@ -106,18 +105,17 @@ _use_auto_optimization=${_use_auto_optimization-y}
 # "full: uses 1 thread for Linking, slow and uses more memory, theoretically with the highest performance gains."
 # "thin: uses multiple threads, faster and uses less memory, may have a lower runtime performance than Full."
 # "none: disable LTO
-_use_llvm_lto=${_use_llvm_lto-none}
+_use_llvm_lto=${_use_llvm_lto-thin}
```

Comment:
Only the default kernel defaults to ThinLTO, the rest stick to GCC

Reply:
Why is this, can you explain? thanks.